giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/service/impl/IdentityProviderRegistry.java

```diff
@@ -34,8 +34,22 @@ public void init() {
     /* (non-Javadoc)
      * @see edu.asu.giles.service.impl.IIdentityProviderRegistry#addProvider(java.lang.String)
      */
+    /**
+     * map providerId to provider name
```

Comment:
this is not what this method is doing. It's registering a new provider with the given authority type.